builtins/amp-img.js

```diff
@@ -255,6 +255,9 @@ export class AmpImg extends BaseElement {
       const {clientWidth, clientHeight} = this.element;
       const widthRatio = naturalWidth / clientWidth;
       const heightRatio = naturalHeight / clientHeight;
+      // TODO: this check can be improved to handle object-fit and transform
```

Comment:
Add either a username or an issue number for the TODO. Note that a username doesn't mean that person has to do the issue, just they have more context (if needed).

Reply:
Done